src/test/java/bowling/frame/EndFrameTest.java

```diff
@@ -0,0 +1,61 @@
+package bowling.frame;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+@DisplayName("마지막 프레임 테스트")
+class EndFrameTest {
+
+	@Test
+	void 마지막_프레임_동등성_테스트() {
+		assertThat(new EndFrame()).isEqualTo(new EndFrame());
+	}
+
+	@Test
+	void 두_번의_시도가_오픈_이면_종료() {
+		EndFrame endFrame = new EndFrame();
+		endFrame.throwBowl(1);
+		endFrame.throwBowl(1);
+
+		assertThat(endFrame.isEnd()).isTrue();
+	}
+
+	@ParameterizedTest(name = " {displayName} : {0}-{1}-{2} =>> {3}")
+	@CsvSource(
+		delimiter = ':',
+		value = {
+			"1:9:1:스페어와 일반점수",
+			"1:9:1:스페어와 스트라이크",
```

Comment:
꼼꼼하게 테스트를 작성해주셨네요 👍👍

> 마지막은 10 아닌가요? :)
```suggestion
			"1:9:10:스페어와 스트라이크",
```

Reply:
앗 오타! 매의 눈 👀  감사드립니다! 👍